src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -1019,10 +1026,10 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
           ...baseRow,
           ...[
             tissueName,
-            tissue ? tissue.logfc : '',
-            tissue ? tissue.ci_r : '',
-            tissue ? tissue.ci_l : '',
-            tissue ? tissue.adj_p_val : '',
+            tissue?.logfc || '',
+            tissue?.ci_r || '',
+            tissue?.ci_l || '',
+            tissue?.adj_p_val || '',
```

Comment:
Was this change intentional? It looks like this may reintroduce the bug fixed in [AG-1347](https://github.com/Sage-Bionetworks/Agora/pull/1283)